central/pruning/pruning.go

```diff
@@ -485,6 +494,127 @@ func (g *garbageCollectorImpl) collectImages(config *storage.PrivateConfig) {
 	}
 }
 
+func (g *garbageCollectorImpl) collectClusters(config *storage.PrivateConfig) {
+	if !features.DecommissionedClusterRetention.Enabled() {
+		return
+	}
+
+	if lastClusterPruneTime.Add(clusterGCFreq).After(time.Now()) {
+		// Only cluster pruning if it's been at least clusterGCFreq since last run
+		return
+	}
+	defer func() {
```

Comment:
It won't actually matter because if it was disabled and then you enabled it you would get 24hrs flex before the clusters get deleted anyway (See line 522).

But yea it does make sense so moving it down